api/src/paths/project/{projectId}/survey/{surveyId}/observation/submission/upload.ts

```diff
@@ -198,14 +191,12 @@ export const insertSurveyOccurrenceSubmission = async (
   surveyId: number,
   source: string,
   inputFileName: string,
-  templateMethodologyId: number | null,
   connection: IDBConnection
 ): Promise<any> => {
   const insertSqlStatement = insertSurveyOccurrenceSubmissionSQL({
     surveyId,
     source,
-    inputFileName,
-    templateMethodologyId
+    inputFileName
```

Comment:
(unrelated to this line)

nit: the JSDoc on line 186 still lists templateMethodologyId.